source/glimmer-components/index.md

```diff
@@ -0,0 +1,37 @@
+## HTML elements as an abstraction for text documents
+
```

Comment:
I like that idea, but I have trouble fitting it into the flow. Since we are primarily targeting new users here, front-loading it at the beginning ("you might not even know what a component is yet, but let me tell you why you would love them and get you to buy in on our philosophy!") seems counterproductive.

Without getting into the specifics of the feature, the "abstraction" part *is* the "why components" and "why this model feels good" at the highest level – this is meant to more than just "ease into" the topic, it is meant to introduce the motivation. DDAU, isolation, etc – all boils down to "abstraction" and "refactorability" (encapsulation) at the end of the day (they tell you what your app is doing without immediately exposing the details, and as a result, you can also refactor the implementation all you want without affect the outside), which is what this section is trying to get at without diving into the specifics.

So I think there are two options:

1. Once the programming model has been fully introduced, it makes sense to call them out explicitly again, either as part of "Composing Components" or breaking it up into two sections ("Composing Components" -> "Thinking in Components" aka Thinking in React).

2. Have a top-level "Thinking in Ember" section at the end of the guides that groups all of these topics (again, once they have finished the tutorials).

Thoughts?